crates/accelerate/src/nlayout.rs

```diff
@@ -91,8 +93,8 @@ impl VirtualQubit {
 #[pyclass(module = "qiskit._accelerate.nlayout")]
 #[derive(Clone, Debug)]
 pub struct NLayout {
-    virt_to_phys: Vec<PhysicalQubit>,
-    phys_to_virt: Vec<VirtualQubit>,
+    pub virt_to_phys: Vec<PhysicalQubit>,
```

Comment:
These are made public to be able to easily call `SparseObservable::apply_layout`, which takes the `virt_to_phys` vector. This could be worked around by using the available public methods, but I think that would require a copy that I wanted to avoid.